ibis/pandas/dispatch.py

```diff
@@ -46,6 +48,15 @@ def pre_execute_default(node, *clients, **kwargs):
     return {}
 
 
+# Merge the results of all client pre-execution with scope
+@pre_execute.register(ops.Node, [ibis.client.Client])
```

Comment:
Should this be:

```
pre_execute.register(ops.Node, ibis.client.Client, [ibis.client.Client])
```
Or are you intentionally matching 0 clients as well?

Reply:
ah, no, i didn't realize it might match 0 clients. what does the function signature look like for that dispatch registration?

Reply:
`_pytest.config.ConftestImportFailure: TypeError("Tried to dispatch on non-type: [<class 'ibis.client.Client'>]\nIn signature: <Node, Client, [<class 'ibis.client.Client'>]>\nIn function: pre_execute",)`

Reply:
function signature looks like `pre_execute(op, client, *clients, **kwargs)`